webrtc-stats.html

```diff
@@ -986,9 +986,9 @@ <h2>
                 <p>
                   Total number of RTP packets received for this <a>SSRC</a>. At the receiving endpoint,
                   this is calculated as defined in [[!RFC3550]] section 6.4.1. At the sending
-                  endpoint the {{packetsReceived}} can be calculated by subtracting the packets lost
-                  from the expected Highest Sequence Number reported in the <a>RTCP Sender Report</a> as
-                  discussed in Appendix A.3. in [[!RFC3550]].
+                  endpoint the {{packetsReceived}} can be calculated by subtracting the Cumulative Number
```

Comment:
On the sender-side the value is an estimate at best.
```suggestion
                  endpoint the {{packetsReceived}} can be estimated by subtracting the Cumulative Number
```